pubspec.yaml

```diff
@@ -120,6 +120,7 @@ flutter:
     - assets/home_screen_icons/
     - assets/topic/
     - assets/topic/asb/
+    - assets/hospital.jpg
```

Comment:
do not add images to top level. Also see that already there are images in topic/asb. Use those instead of adding new one

Reply:
I need this image for the experiment purpose because I know the offset, once my logic is working fine will add more images, I move it to topics.